.github/workflows/post_merge.yaml

```
@@ -14,7 +14,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        flutter-version: [ '3.24.3', '3.27.3' ]
+        flutter-version: [ '3.24.3', '3.27.4', '3.29.3', '3.32.7' ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)